ApiBundle/Tests/Transformer/ContentTransformerTest.php

```diff
@@ -60,21 +52,44 @@ public function setUp()
      */
     public function testTransform()
     {
-        $facade = Phake::mock('OpenOrchestra\BaseApi\Facade\FacadeInterface');
-        $facade->label = 'draft';
-        $facade->name = 'fakeName';
-        $facade->value = 'fakeValue';
-
+        $content = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentInterface');
+        Phake::when($content)->getId()->thenReturn(1);
+        $contentData = 'contentData';
+        $contentDate = '2015/09/01';
+        $contentBool = true;
         $attribute = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentAttributeInterface');
-        Phake::when($this->content)->getAttributes()->thenReturn(array($attribute, $attribute));
-
-        Phake::when($this->transformer)->transform(Phake::anyParameters())->thenReturn($facade);
-        Phake::when($this->transformerAttribute)->transform(Phake::anyParameters())->thenReturn($facade);
-
-        $facade = $this->contentTransformer->transform($this->content);
+        Phake::when($content)->getAttributes()->thenReturn(array($attribute, $attribute));
+        Phake::when($content)->getContentId()->thenReturn($contentData);
+        Phake::when($content)->getContentType()->thenReturn($contentData);
+        Phake::when($content)->getName()->thenReturn($contentData);
+        Phake::when($content)->getVersion()->thenReturn($contentData);
+        Phake::when($content)->getContentTypeVersion()->thenReturn($contentData);
+        Phake::when($content)->getLanguage()->thenReturn($contentData);
+        Phake::when($content)->isDeleted()->thenReturn($contentBool);
+        Phake::when($content)->isLinkedToSite()->thenReturn($contentBool);
+        Phake::when($content)->getCreatedAt()->thenReturn($contentDate);
+        Phake::when($content)->getUpdatedAt()->thenReturn($contentDate);
```

Comment:
Here too you should use different values instead of a generic one for all getters